### ISSUE TITLE
Can you change to dont need to always  press  button to send 

### ISSUE THREAD

**Issue by joe12801** [cogentapps/chat-with-gpt#126]
would you change to when I finish my speaking can auto to send message,
dont need to always press button to send
just like this google extension

https://chrome.google.com/webstore/detail/talk-to-chatgpt/hodadfhfagpiemkeoliaelelfbboamlk?hl=en

**Comment by lee-b** [comment]
Yes please.  It should auto-detect when the user is speaking, and send when they stop, allowing a full, two-way, hands-free conversation.

**Comment by aghanki** [comment]
I think it would be awesome if you could find a way to communicate hands-free, without the need to press the Enter button.
